tests/func/test_datachain.py

```diff
@@ -1004,7 +1004,7 @@ def name_len_interrupt(_name):
         DataChain.from_storage(cloud_test_catalog_tmpfile.src_uri, session=session)
         .filter(C("file.size") < 13)
         .filter(C("file.path").glob("cats*") | (C("file.size") < 4))
-        .settings(parallel=2, workers=2)
+        .settings(parallel=2, workers=1)
```

Comment:
Suggested solution 1: use a single worker, so it's always the local one, and we can capture its output.